Apps/frontend/src/shader/indicatorShader.js

```diff
@@ -0,0 +1,31 @@
+export const fragmentShader = `#version 300 es
```

Comment:
Nothing big, but since both shaders for the indicatorLines are here I'd prefer indicatorShaders.